op-scim-bridge/chart/op-scim-bridge-mp/Chart.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: op-scim-bridge
   repository: https://1password.github.io/op-scim-helm/
-  version: 2.10.1
+  version: 2.10.3
 - name: common
   repository: https://charts.bitnami.com/bitnami
   version: 1.17.1
-digest: sha256:dd5c3f18d7a7b070e36555c754da14064afd7bf2f69d66f9265ded11531907ce
-generated: "2023-05-09T08:48:21.931439-04:00"
```

Comment:
Does this generated value need to be updated in this MR?

Reply:
Yes, it references the upstream chart from `op-scim-helm` and needs to be updated with each version bump.